ui/apps/everest/.e2e/release/restore-new-cluster.e2e.ts

```diff
@@ -0,0 +1,534 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { expect, test } from '@playwright/test';
+import {
+  deleteDbCluster,
+  gotoDbClusterBackups,
+  gotoDbClusterRestores,
+  restartDbCluster,
+  resumeDbCluster,
+  suspendDbCluster,
+} from '@e2e/utils/db-clusters-list';
+import { getTokenFromLocalStorage } from '@e2e/utils/localStorage';
+import { getClusterDetailedInfo } from '@e2e/utils/storage-class';
+import {
+  moveForward,
+  submitWizard,
+  populateBasicInformation,
+  populateResources,
+  populateAdvancedConfig,
+} from '@e2e/utils/db-wizard';
+import {
+  fillScheduleModalForm,
+  ScheduleTimeOptions,
+} from '@e2e/pr/db-cluster/db-wizard/db-wizard-utils';
+import { EVEREST_CI_NAMESPACES } from '@e2e/constants';
+import {
+  waitForStatus,
+  waitForDelete,
+  findRowAndClickActions,
+} from '@e2e/utils/table';
+import { clickCreateSchedule } from '@e2e/pr/db-cluster-details/utils';
+import { prepareTestDB, dropTestDB, queryTestDB } from '@e2e/utils/db-cmd-line';
+import { getDbClusterAPI } from '@e2e/utils/db-cluster';
+import { shouldExecuteDBCombination } from '@e2e/utils/generic';
+
+let token: string;
+
+test.describe.configure({ retries: 0 });
+
+function getNextScheduleMinute(incrementMinutes: number): string {
+  const d: number = new Date().getMinutes();
+  const minute: number = (d + incrementMinutes) % 60;
+
+  return minute.toString();
+}
+
+[
+  { db: 'psmdb', size: 3 },
+  { db: 'pxc', size: 3 },
+  { db: 'postgresql', size: 3 },
+].forEach(({ db, size }) => {
+  test.describe(
+    'Restore to a new cluster',
+    {
+      tag: '@release',
+    },
+    () => {
+      test.skip(!shouldExecuteDBCombination(db, size));
+      test.describe.configure({ timeout: 720000 });
+
+      const clusterName = `${db}-${size}-schbkp`;
```

Comment:
Please correct the cluster name to something unique to this test, maybe `${db}-${size}-rest-new`.